src/main/java/DukeTask/Task.java

```diff
@@ -0,0 +1,41 @@
+package DukeTask;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
+        return (isDone ? "X": " ");
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(){
+        this.isDone = true;
+    }
+
+    public void setUndone(){
+        this.isDone = false;
+    }
+
+    /*public String printTask(){
+        return ("["+getStatusIcon()+"] "+description);
+    }
+     */
+
+    @Override
+    public String toString(){
+        return ("["+getStatusIcon()+"] "+description);
+    }
```

Comment:
Consider standardizing the spaces before and after operators (e.g "[" + getStatusIcon() + "] " + description)

Reply:
Thank you for your suggestion. I have updated my code! Thank you!